c2nim.nim

```diff
@@ -52,6 +56,7 @@ Options:
                          for example `--mangle:'{u?}int{\d+}_t=$1int$2'` to
                          convert C <stdint.h> to Nim equivalents
                          (multiple --mangle options are supported)
+  --stdints              Mangle C stdint's into Nim style int's
```

Comment:
this would feel more natural as a `--no-stdints` - ie the quality of the wrapper is significantly improved when these types are recognised and it's hard to imagine code that uses these names for anything else than the standard interpretation